packages/css/dropdownmenu.css

```diff
@@ -1,10 +1,6 @@
 .ds-dropdownmenu {
-  --dsc-dropdownmenu-background: var(--ds-color-neutral-background-default);
-  --dsc-dropdownmenu-padding: var(--ds-spacing-2);
-  --dsc-dropdownmenu-border-radius: min(1rem, var(--ds-border-radius-md));
-  --dsc-dropdownmenu-border: 1px solid var(--ds-color-neutral-border-subtle);
+  --dsc-dropdownmenu-padding: var(--ds-spacing-3) var(--ds-spacing-2);
   --dsc-dropdownmenu-min-width: 260px;
-  --dsc-dropdownmenu-shadow: var(--ds-shadow-md);
   --dsc-dropdownmenu-hr-border-color: var(--ds-color-neutral-border-subtle);
```

Comment:
Maybe we should have a convention here - `--dsc-dropdown-divider` like in `--dsc-table-header-divider` or something? Also, there are no stories showing usage of hr in Dropdown? Maybe we even can remove `--dsc-dropdownmenu-hr-border-color` entirely now that `Divider` is subtle by default? ☺️

Reply:
up to @Thunear to decide here, I don't think we need this anymore either

Reply:
I agree. We should use the `Divider` component inside these types of components.